ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,2 @@
+<div class="alert alert-danger" role="alert" ng-repeat="error in errors"><span class="fa fa-warning"></span> {{error}}</div>
+<a href="#/comparechooser?originalProject={{originalProject}}&originalRevision={{originalRevision}}&newProject={{newProject}}&newRevision={{newRevision}}">Modify choice</a>
```

Comment:
OK, the last result is looks like this @tojonmz @wlach 
<img width="1265" alt="screen shot 2015-07-31 at 3 01 04 pm" src="https://cloud.githubusercontent.com/assets/2166227/9002322/0c6e1cd8-3795-11e5-9037-7a6aa585ce2a.png">